tests/factories.py

```diff
@@ -2,135 +2,167 @@
 """Factory classes for easily generating test objects."""
 
 from __future__ import absolute_import
+from __future__ import unicode_literals
 
 import random
-import datetime
 
 import factory
+import faker
 
-from h.api import storage
+from h import db
+from h import models
 from h.accounts import models as accounts_models
```

Comment:
`accounts_models`? It's used by the `User` class at the bottom of the file, which this pr hasn't touched